app/frontend/src/features/BugReport.tsx

```diff
@@ -86,6 +102,17 @@ export default function BugReport() {
 
   return (
     <>
+      {bug && (
+        <SuccessSnackbar>
+          <>
+            {BUG_REPORT_SUCCESS}
+            <Link href={bug.bugUrl} target="_blank">
```

Comment:
```suggestion
            <Link href={bug.bugUrl}>
```
I don't think this needs to be opened in a new tab/window? It's bad for accessibility to do this by default without letting the user know in some way.